--- jslint.js.orig	2011-11-18 08:22:50.000000000 -0600
+++ jslint.js	2011-11-18 08:23:14.000000000 -0600
@@ -1159,7 +1159,9 @@
             html: hx,
             style: sx,
             styleproperty: ssx
-        };
+        },
+
+        shouldMethods = ['exist'];
 
 
     function F() {}     // Used by Object.create
@@ -3147,7 +3149,9 @@
                 } else if (!the_statement.assign &&
                         the_statement.id !== 'delete' &&
                         the_statement.id !== '++' &&
-                        the_statement.id !== '--') {
+                        the_statement.id !== '--' &&
+                        (!the_statement.second || 
+                         shouldMethods.indexOf(the_statement.second.string) < 0)) {
                     warn('assignment_function_expression', token);
                 }
                 semicolon();
