--- jslint.js.orig	2011-07-21 08:41:41.000000000 -0500
+++ jslint.js	2011-07-21 08:42:14.000000000 -0500
@@ -932,7 +932,9 @@
             html: hx,
             style: sx,
             styleproperty: ssx
-        };
+        },
+
+        shouldMethods = ['exist'];
 
     function F() {}
 
@@ -2539,7 +2541,7 @@
 // Look for the final semicolon.
 
         if (!t.block) {
-            if (!r || !r.exps) {
+            if ((!r || !r.exps) && shouldMethods.indexOf(r.right) < 0) {
                 warning(
 "Expected an assignment or function call and instead saw an expression.",
                         token);
